### DISCUSSION TITLE
The version of index 'xxx.obf' is not supported

### DISCUSSION THREAD

**Discussion by Hahguyua** [osmandapp/OsmAnd#12770] [Q&A]
Hey osmands,
I am having trouble using osmand app since a while. It didn't want to install some certain new maps. Seemed like it depended on the size of the file.
So I tried deleting and reinstalling. Now none of the maps at all are loaded, not even the world map. I tried with an older version, same problem.
I am using Samsung sm A500 with lineage, android 9

I am posting here because I don't have telegram. I hope its fine!
Thx

**Comment by vshcherb** [comment]
Version of obf files didn't change last 5 years, so it's likely that files themselves are corrupted and you need to redownload them. If you use external SD card, try to use internal storage to test it.

**Reply by Hahguyua** [reply]
Changing to infernal storage didnt help.
I am using osmand only since a few months vor so, so sureley this isnt the reason.

**Reply by Hahguyua** [reply]
I tried many times to download different files, also downloaded them via the browser, all same
